src/client/options/mod.rs

```diff
@@ -289,54 +289,54 @@ pub struct ClientOptions {
     /// The application name that the Client will send to the server as part of the handshake. This
     /// can be used in combination with the server logs to determine which Client is connected to a
     /// server.
-    #[builder(default)]
+    #[builder(default, setter(strip_option))]
     pub app_name: Option<String>,
 
-    #[builder(default)]
+    #[builder(default, setter(skip))]
```

Comment:
Since `heartbeat_freq_test` has the `#[cfg(test)]` attribute, it only gets compiled when the tests are being run. The builder appears to inherit this as well from testing it out. For the non-`#[cfg(test)]` ones, they actually do get included in the public builder, regardless of the field's ACL. In fact, the 1.x driver actually leaks these :grimacing: (https://docs.rs/mongodb/1.2.1/mongodb/options/struct.ClientOptions.html#method.builder)